internal/app/admin/http/handler_test.go

```diff
@@ -199,6 +199,108 @@ func TestAdminServer_CacheDumpHandler_NotFound(t *testing.T) {
 	assert.Equal(t, "", rr.Body.String())
 }
 
+func TestAdminServer_ClearCacheHandler(t *testing.T) {
```

Comment:
what is the difference between tests here and in `internal/app/admin/http/cache_test.go` ?

Reply:
These tests focus on non-wildcard cases of clearing the cache. I originally left these in `internal/app/admin/http/handler_test.go` since that seemed to be what happened with the cache dump handler tests in a previous PR that split off some cache dump tests into their own file (TestAdminServer_CacheDumpHandler and TestAdminServer_CacheDumpHandler_NotFound remained in handler_test.go while all other cache dump tests were moved to `internal/app/admin/http/cache_test.go`. I went ahead and moved all cache related tests for both admin handlers from the handler file and into the cache file.